src/Perl6/World.nqp

```diff
@@ -5158,6 +5185,8 @@ class Perl6::World is HLL::World {
         # Make sure it's not an empty name.
         unless +@name { nqp::die("Cannot look up empty name"); }
 
+        self."!no_experimental_rakuast"(@name);
```

Comment:
I haven't noticed any significant degradation on core compile time. Besides, there are more concerning techniques like further down this method where `try { self.find_symbol }` is used. Exception handling it self is way more costly. Moreover, in one location in `Actions.nqp` a call to `is_name` is immediately followed by `find_symbol` which is, essentially, nothing else but `find_symbol` duplicated. And that is not to mention that `Grammar.nqp` is using it all over the place and often corresponding `Action` methods use `find_symbol` too – i.e. it would be cases of the same duplication.